front/app/services/homepageSettings.ts

```diff
@@ -1,10 +1,21 @@
 import { API_PATH } from 'containers/App/constants';
```

Comment:
[Created a ticket ](https://citizenlab.atlassian.net/browse/CL-1138)for this file, still a few things to work out at the end.